cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -130,122 +134,124 @@ func main() {
 		fmt.Println("timestamp,session,segment,seg_dur,transcode_time")
 	}
 
-	segCount := 0
-	realTimeSegCount := 0
-	srcDur := 0.0
 	var mu sync.Mutex
-	transcodeDur := 0.0
-	for i := 0; i < *concurrentSessions; i++ {
-		wg.Add(1)
-		go func(k int, wg *sync.WaitGroup) {
-			var tc *ffmpeg.Transcoder
-			if *detectionFreq > 0 {
-				t := time.Now()
-				tc, err = ffmpeg.NewTranscoderWithDetector(detectionOpts.Detector, devices[k%len(devices)])
-				end := time.Now()
-				fmt.Printf("InitDetectorSession time %0.4v for session %v\n", end.Sub(t).Seconds(), i)
-				if err != nil {
-					glog.Fatalf("Could not initialize detector")
-				}
-			} else {
-				tc = ffmpeg.NewTranscoder()
-			}
-			for j, v := range pl.Segments {
-				iterStart := time.Now()
-				if *segs > 0 && j >= *segs {
-					break
-				}
-				if v == nil {
-					continue
-				}
-				u := path.Join(dir, v.URI)
-				in := &ffmpeg.TranscodeOptionsIn{
-					Fname: u,
-					Accel: accel,
-				}
-				if ffmpeg.Software != accel {
-					in.Device = devices[k%len(devices)]
+	for repeat_counter := 0; repeat_counter < *repeat; repeat_counter++ {
```

Comment:
Nit: Suggest using camel case here to follow Go idiom.

Reply:
Fixed